src/test/java/com/yunhalee/walkerholic/activity/service/ActivityServiceTests.java

```diff
@@ -28,48 +30,52 @@
     @Autowired
     ActivityRepository activityRepository;
 
+    @Value("${AWS_S3_BUCKET_URL}")
+    private String AWS_S3_BUCKET_URL;
+
     @Test
-    public void createActivity() {
+    public void createActivity() throws IOException {
         //given
         String name = "testActivity";
         Integer score = 1;
         String description = "This is test Activity.";
 
-        ActivityCreateDTO activityCreateDTO = new ActivityCreateDTO(name, score, description);
+        ActivityRequest activityRequest = new ActivityRequest(name, score, description);
         MultipartFile multipartFile = new MockMultipartFile("uploaded-file",
             "sampleFile.txt",
             "text/plain",
             "This is the file content".getBytes());
         //when
-        ActivityCreateDTO activityCreateDTO1 = activityService
-            .createActivity(activityCreateDTO, multipartFile);
+        ActivityResponse activityResponse1 = activityService
+            .create(activityRequest, multipartFile);
 
         //then
-        assertNotNull(activityCreateDTO1.getId());
-        assertEquals(name, activityCreateDTO1.getName());
-        assertEquals(score, activityCreateDTO1.getScore());
-        assertEquals(description, activityCreateDTO1.getDescription());
-        assertEquals("/activityUploads/" + activityCreateDTO1.getId() + "/" + "sampleFile.txt",
-            activityCreateDTO1.getImageUrl());
+        assertNotNull(activityResponse1.getId());
+        assertEquals(name, activityResponse1.getName());
+        assertEquals(score, activityResponse1.getScore());
+        assertEquals(description, activityResponse1.getDescription());
+        assertTrue(activityResponse1.getImageUrl()
+            .contains(AWS_S3_BUCKET_URL + "activityUploads/" + activityResponse1.getId() + "/"));
```

Comment:
```suggestion
            .contains(bucketUrl + "activityUploads/" + activityResponse1.getId() + "/"));
```

상수가 맞나요? 
그리고 상수 값은 static final이 관례입니다.

> https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

Reply:
AWS_S3_BUCKET_URL은 value애노테이션을 통하여 properties에 작성된 값을 불러오도록 설정하였습니다!

혹시 value애노테이션으로 불러온 값도 final 설정이 필요할까요? 😀

🙌 감사합니다!! 🙌

Reply:
properties에서 불러온 값을 변경할 일이 있을까요? 가변데이터로 관리하지 않기 위해 properties 로 관리한다고 생각해요.
그런 의미에서 final은 필요하다고 생각해요. 
그리고 위의 피드백은 상수가 아닌데, 대문자 + snake case 라 피드백 드린 거였어요.

Reply:
확인하여 변경하였습니다!